internal/translation/deployment/deployment.go

```diff
@@ -27,24 +28,33 @@ type AtlasDeploymentsService interface {
 	DeleteDeployment(ctx context.Context, deployment Deployment) error
 	ClusterWithProcessArgs(ctx context.Context, cluster *Cluster) error
 	UpdateProcessArgs(ctx context.Context, cluster *Cluster) error
+
+	GetCustomZones(ctx context.Context, projectID, clusterName string) (map[string]string, error)
+	CreateCustomZones(ctx context.Context, projectID, clusterName string, mappings []akov2.CustomZoneMapping) (map[string]string, error)
+	DeleteCustomZones(ctx context.Context, projectID, clusterName string) error
+	GetZoneMappingMap(ctx context.Context, projectID, deploymentName string) (map[string]string, error)
```

Comment:
Simplify, reduce rendundancy naming:

```suggestion
	GetZoneMapping(ctx context.Context, projectID, deploymentName string) (map[string]string, error)
```